### PR TITLE
Further code improvements

### PR DESCRIPTION
**Pull request by AndyScherzinger**
random fixes for spotbugs, lombok usage, cleanups, etc.

### CODE REVIEW

File: src/main/java/com/owncloud/android/operations/RenameFileOperation.java [hunk 1]
```diff
@@ -42,10 +44,10 @@
 
     private static final String TAG = RenameFileOperation.class.getSimpleName();
 
-    private OCFile mFile;
-    private String mRemotePath;
-    private String mNewName;
-    private String mNewRemotePath;
+    @Getter
+    private OCFile file;
```
Comment (tobiasKaminsky) [inline src/main/java/com/owncloud/android/operations/RenameFileOperation.java]: one line?

File: src/test/java/com/owncloud/android/utils/PowerUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/*
```
Comment (tobiasKaminsky) [inline src/test/java/com/owncloud/android/utils/PowerUtilsTest.java]: Should this be here? :thinking:

Comment (AndyScherzinger) [reply src/test/java/com/owncloud/android/utils/PowerUtilsTest.java]: Nope, fixed via a new rebase against master 👍

File: src/main/java/com/owncloud/android/operations/RenameFileOperation.java (hunk 1 above)
Comment (AndyScherzinger) [reply src/main/java/com/owncloud/android/operations/RenameFileOperation.java]: fixed via https://github.com/nextcloud/android/pull/3873/commits/7af55d43be0d21a4c6ebb761db1fa9302e0ee251

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky Thanks for the review. All your remarks have been fixed :heart:

**Comment by tobiasKaminsky** [comment]
Now codacy is complaining: https://app.codacy.com/app/nextcloud/android/pullRequest?prid=3413801

(I manually changed from dan0xii to nextcloud)

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky fun fact: https://app.codacy.com/app/Nextcloud/android/pullRequest?prid=3413731 this is what I see for the PR/branch...

**Comment by tobiasKaminsky** [comment]
Codacy is acting crazy…

**Comment by AndyScherzinger** [comment]
> Codacy is acting crazy…

Yeah, strange. the codacy score should basically stay more or less the same since most stuff is cleanup and renames which let's codacy detect the exact same issue that have been there before, only the variable has a different name now...

**Comment by AndyScherzinger** [comment]
Merged, Codacy was finally happy 🎉

**Pull request merged**